Command/ReverseCommand.php

```diff
@@ -63,7 +63,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $filesystem = new Filesystem();
             $generated  = $this->getCacheDir().'/schema.xml';
             $filename   = $name . '_reversed_schema.xml';
-            $destFile   = $this->getApplication()->getKernel()->getRootDir() . '/propel/generated-schemas/' . $filename;
+            $destFile   = $this->getApplication()->getKernel()->getProjectDir() . '/var/propel/generated-schemas/' . $filename;
```

Comment:
Is this change in location intentional?
`./app/propel/generated-schemas/` --> `./var/propel/generated-schemas/`

Reply:
It is, seemed better to me to group generated files together in a subdir than in the root dir.